frontend/src/static/css/style.scss

```diff
@@ -253,14 +253,35 @@ header {
     padding-right: 10px;
   }
 
+  .mui-btn {
+    bottom: -8px;
+    box-shadow: none;
+    color: mui-color('grey');
+    font-size: .75rem;
+    font-weight: bold;
+    height: 15px;
+    left: -8px;
+    line-height: 5px;
+    margin: 0;
+    padding: 0;
```

Comment:
Perhaps the 
* bottom: -8px;
* height: 15px;
* line-height: 5px; 

Can be just changed to 
* vertical-align: middle;
* background: transparent;